README.md

```diff
@@ -80,6 +80,10 @@ The Framework Laptop 16 has a protection scheme in place to prevent Input Module
 Module detection is done using the BOARD_ID pin. It is possible to override this setting on the system, but at the risk of shorting the system
 or modules.
 
+Note that although a "keyboard size" module has pads on the left and right
```

Comment:
What do you mean? It does not have pads on both sides.

It may look like the pad could be on the either side from a mechanical point of view when looking at the keyboard. But nothing about the system is designed to handle that (neither firmware, mechanical or electrical).
There also isn't an easy way you could mod the keyboard to have the pads on the other side, so I don't think it's accurate to say there are pads on both sides.

Maybe we could add that note to the mechanical documentation? The only case where you need to know about this, is when you want to build your own module.
But really on the other side of the module there isn't enough space to put a microcontroller and pads :sweat_smile: